php-checks/src/test/java/org/sonar/php/checks/wordpress/WordPressConfigNameCheckTest.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * SonarQube PHP Plugin
+ * Copyright (C) 2010-2021 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+package org.sonar.php.checks.wordpress;
+
+import org.junit.Test;
+import org.sonar.plugins.php.CheckVerifier;
+
+public class WordPressConfigNameCheckTest {
+
+  @Test
+  public void test() {
+    CheckVerifier.verify(new WordPressConfigNameCheck(), "wordpress/WordPresConfigNameCheck/wp-config.php");
```

Comment:
Any reason to not use the wordPressVerifier?

Now it doesn't matter, since you've already made the effort.
Can't check. I'll take it that way.

Reply:
No strong reason, no.

My thought was either I create one additional folder, or:
- Import a new class
- Make my test class extend that class
- Add `throws IOException` to the tests methods.

So I went for the first solution :smile: . I think using the WordPress verifier makes sense to avoid creating many new folders, and/or we test absence.